src/main/java/core/model/player/Player.java

```diff
@@ -1712,7 +1757,7 @@ public int getValue4Skill(int skill) {
             case WINGER -> m_iFluegelspiel;
             case SCORING -> m_iTorschuss;
             case SET_PIECES -> m_iStandards;
-            case STAMINA -> m_iKondition;
+            case STAMINA -> getStamina();
```

Comment:
in Player.calcsubskill you mention that stamina is the subskill of stamina/kondition (german word for stamina). but here you return it as major value of stamina. major value of stamina is downloaded from hattrick and does not need to be calculated.